### DISCUSSION TITLE
How to create sheets to group fields in attribute editing mode

### DISCUSSION THREAD

**Discussion by Lorenzofrangini** [opengisch/QField#5263] [Q&A]
Hi, I have vector layer with several columns. When I'm in the editing mode and I add a new feature as always it appears a window with all my columns to be filled with data. Since I have several columns I want to group them using the sheet visualization like those in several layers of sample projects (e.g., bees farming) as displayed below and highlighted with yellow circles.

![attribute_table](https://github.com/opengisch/QField/assets/170432020/88426582-627e-49e5-8f44-36980f882364)

I cannot find how to create this visualization in the how-to-guide of QField website. I suppose I need to do it on QGIS Desktop, and I'm using QGis 3.34.6.

Thanks
Lorenzo

**Comment by Ipickedausername** [comment]
Hi, 

You can go to the layer's properties 
> forms 
> choose "drag-and-drop design" in the dropdown menu
> Add a new tab or group to the form (green +)
> Choose "Tab" and give it the name you want. 

![Picture3](https://github.com/opengisch/QField/assets/72577399/f2caba30-f8a3-4d01-bab1-b584a79ded6d)

**Reply by Lorenzofrangini** [reply]
Solved! :)
